test/quantization/core/test_quantized_op.py

```diff
@@ -4750,7 +4772,91 @@ def test_qconv2d(
                 input_channels_per_group, (height, width),
                 output_channels_per_group, groups, kernels, strides, pads, None,
                 dilations, X_scale, X_zero_point, W_scale, W_zero_point,
-                Y_scale, Y_zero_point, use_bias, use_relu, use_channelwise, False, input_dtype=X_qdtype, output_dtype=X_qdtype)
+                Y_scale, Y_zero_point, use_bias, post_op, use_channelwise, False, input_dtype=X_qdtype, output_dtype=X_qdtype)
+
+    """Tests the correctness of quantized convolution op."""
+    @given(batch_size=st.integers(1, 3),
+           input_channels_per_group=st.sampled_from([2, 4, 5, 8, 16, 32]),
+           height=st.integers(10, 16),
+           width=st.integers(7, 14),
+           output_channels_per_group=st.sampled_from([2, 4, 5, 8, 16, 32]),
+           groups=st.integers(1, 10),
+           kernel_h=st.integers(1, 7),
+           kernel_w=st.integers(1, 7),
+           stride_h=st.integers(1, 2),
+           stride_w=st.integers(1, 2),
+           pad_h=st.integers(0, 2),
+           pad_w=st.integers(0, 2),
+           dilation=st.integers(1, 2),
+           X_scale=st.floats(1.2, 1.6),
+           X_zero_point=st.integers(0, 4),
+           W_scale=st.lists(st.floats(0.2, 1.6), min_size=1, max_size=2),
+           W_zero_point=st.lists(st.integers(-5, 5), min_size=1, max_size=2),
+           Y_scale=st.floats(4.2, 5.6),
+           Y_zero_point=st.integers(0, 4),
+           use_bias=st.booleans(),
+           post_op=st.sampled_from(["add"]),
+           use_channelwise=st.booleans(),
+           X2_scale=st.floats(1.2, 1.6),
+           X2_zero_point=st.integers(0, 4),)
+    @skipIfNoONEDNN
+    def test_qconv2d_add(
+            self,
+            batch_size,
+            input_channels_per_group,
+            height,
+            width,
+            output_channels_per_group,
+            groups,
+            kernel_h,
+            kernel_w,
+            stride_h,
+            stride_w,
+            pad_h,
+            pad_w,
+            dilation,
+            X_scale,
+            X_zero_point,
+            W_scale,
+            W_zero_point,
+            Y_scale,
+            Y_zero_point,
+            use_bias,
+            post_op,
+            use_channelwise,
+            X2_scale,
+            X2_zero_point
+    ):
+        with override_quantized_engine('onednn'):
+            input_channels = input_channels_per_group * groups
+            output_channels = output_channels_per_group * groups
+            kernels = (kernel_h, kernel_w)
+            strides = (stride_h, stride_w)
+            pads = (pad_h, pad_w)
+            dilations = (dilation, dilation)
+
+            qconv = torch.ops.quantized.conv2d
+            if post_op == "add":
+                qconv = torch.ops.quantized.conv2d_add
```

Comment:
if this is only "add" we can remove the post_op argument

Reply:
Thanks for the suggestions, I have removed the post_op argument and the check. In next PR, I will put conv2d_add_relu into   a separate test.